### PR TITLE
MRP feedback controller tuning

### PR DESCRIPTION
**Pull request by galensavidge**
In this MR:
1. A page for the MRP controller and tuning process;
2. A new script that calculates gains;
3. Changes to the feedback controller module to take gain matrices instead of scalars.

I still need to play with the gains a bit more to get better performance.

### CODE REVIEW

File: aces/scripts/generateControlGains.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    K = np.max(Ki)
+    print(f'{K=}')
+
+    z3 = np.zeros((3, 3))
+    Iinv = np.linalg.inv(I_rw)
+
+    # Pick an integral gain K_I that's "small" relative to P
+    K_I = Iinv @ P / 10
+    #  K_I = Iinv @ P * 5
```
Comment (alboehm) [inline aces/scripts/generateControlGains.py:53]: what's the commented out line for? Were you just trying it out?

File: aces/docs/source/algorithms/mrpfeedbackcontroller.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+time :math:`T_s` and damping ratio :math:`\xi`. We will start off by making two
+simplifying assumptions:
+
+#. The inertia matrix :math:`I_{RW}` is diagonal with entries :math:`I_i`;
+
+#. There is no integral feedback.
+
+We can make use of the closed-loop error dynamics for this feedback controller
+developed in Schaub section 8.4.3. Rearranging Schaub eq. 8.119 we get an
+equation for :math:`P_i`, the diagonals of the :math:`P` matrix:
```
Comment (alboehm) [inline aces/docs/source/algorithms/mrpfeedbackcontroller.rst:45]: can you specify the P, K, and K_I variables as proprotional derivative and integral somewhere in this doc.

File: aces/docs/source/algorithms/mrpfeedbackcontroller.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+   \end{bmatrix}
+   \begin{bmatrix}
+      \boldsymbol{\sigma} \\
+      \delta\boldsymbol{\omega} \\
+      \boldsymbol{z}
+   \end{bmatrix}
+
+Substituting in the previously computed gain matrices into this linear matrix
+equation lets us test the error dynamics of the closed-loop system with the
+integral feedback terms.
```
Comment (alboehm) [inline aces/docs/source/algorithms/mrpfeedbackcontroller.rst:115]: add a quick line about iterating on the controller design with the integral term, settling time, and damping ratio

Comment (galensavidge) [reply aces/docs/source/algorithms/mrpfeedbackcontroller.rst:115]: Good idea

File: aces/scripts/generateControlGains.py (hunk 1 above)
Comment (galensavidge) [reply aces/scripts/generateControlGains.py]: Yeah. I removed it

File: aces/docs/source/algorithms/mrpfeedbackcontroller.rst (hunk 2 above)
Comment (galensavidge) [reply aces/docs/source/algorithms/mrpfeedbackcontroller.rst:93]: I copied the controller equations from the BSK doc. Seems nice to have here for context

File: aces/docs/source/algorithms/mrpfeedbackcontroller.rst (hunk 3 above)
Comment (galensavidge) [reply aces/docs/source/algorithms/mrpfeedbackcontroller.rst]: Done

**Comment by galensavidge** [comment]
I fixed the unit test as well

**Review by alboehm — APPROVED** [review]

**Pull request merged**